tldr.py

```diff
@@ -160,6 +160,7 @@ def get_page_for_platform(
         try:
             data = urlopen(
                 Request(page_url, headers=REQUEST_HEADERS),
+                timeout = 2,
```

Comment:
```suggestion
                timeout=2,
```

Reply:
Is two seconds long enough?

Reply:
> Is two seconds long enough?

I think so, but i wouldn't care if it was set to 10 seconds or something like that. The problem that i saw was that, if no timeout was specified, there were some occasions where the program would get stuck for more than a minute until throwing a timeout error. However I believe that 2 seconds are more than enough.

Reply:
I've bumped this up to 10 seconds as I feel like while 2 is _probably_ fine, better to be a touch conservative and hopefully avoid any people complaining about things "suddenly" breaking on them.